libs/utils/src/elections.ts

```diff
@@ -1,7 +1,7 @@
 import { Optional, unique } from '@votingworks/basics';
 import { Election } from '@votingworks/types';
 
-export function getElectionSheetCount(election: Election): Optional<number> {
+export function getMaxSheetsPerBallot(election: Election): Optional<number> {
```

Comment:
Should this function just take the maximum sheet number value from the grid layouts instead of counting the length of unique sheet numbers?